bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -304,8 +304,11 @@ thingName, getThing().getLabel(), thingType, config.deviceAddress.toUpperCase(),
         }
 
         if (profile.alwaysOn || !profile.isInitialized()) {
-            updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.CONFIGURATION_PENDING,
-                    messages.get("status.unknown.initializing"));
+            ThingStatusDetail detail = getThingStatusDetail();
+            if (detail != ThingStatusDetail.DUTY_CYCLE) {
+                updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.CONFIGURATION_PENDING,
```

Comment:
```suggestion
                updateStatus(ThingStatus.ONLINE, ThingStatusDetail.CONFIGURATION_PENDING,
```

See https://www.openhab.org/docs/concepts/things.html#status-details

Reply:
hmm, but at that tie the thing is not ONLINE = functional
Maybe INITIALIZING / CONFIG_PENDING?

Reply:
changed, please check